lib/spack/docs/environments.rst

```diff
@@ -424,32 +424,34 @@ file may also be copied out of the environment, renamed, etc.
 Included Concrete Environments
 ------------------------------
 
-Spack environments can create an evironment based off of information in already
+Spack environments can create an environment based off of information in already
 established environments. You can think of it as a combination of existing
-environnments. It will gathering information from the existing environment's
+environments. It will gather information from the existing environment's
 ``spack.lock`` and use that during the creation of this included concrete
-environmenit. When an included concrete environment is create it will generate
-a ``spack.lock`` file.
+environment. When an included concrete environment is created it will generate
+a ``spack.lock`` file for the newly created environment.
 
 
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Creating included environment
+Creating included environments
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-To create an included concrete environment, you must have at least one exsiting
+To create a combined concrete environment, you must have at least one exsiting
 concrete environment. You will use the command ``spack env create`` with the
 argument ``--include-concrete`` followed by the name or path of the environment
-you'd like to include.
+you'd like to include. Here is an example of how to create a combined environment
+from the command line.
 
-.. code:: console
+.. code-block:: console
 
    $ spack env create myenv
    $ spack -e myenv add python
    $ spack -e myenv concretize
    $ spack env create --include-concrete myenv included_env
 
 
-You can also create an included environment by adding the ``include_concrete``
-heading in the yaml.
+You can also create an included environment through the ``spack.yaml`` file. It
```

Comment:
Should this be "create a combined environment using the" or "You can also include an environment directly in the..."?